### PR TITLE
[Duplicate of #5365] Introduction to Kernel Methods in Machine Learning

### LINKED ISSUE #4567: Introduction to Kernel Methods in Machine Learning
#### Proposed article introduction
Kernel method in machine learning is defined as a class of pattern analysis algorithms that are used to study and find general types of relations (such as correlation, classification, ranking, clusters, principle components, and so on) in datasets by explicitly transforming raw data representation into feature vector representation using a user-specified feature map, allowing the high dimensional implicit feature space of these data to be operated with computer systems.

#### Key takeaways 
Principle components analysis
Support vector machine
spectral clustering

#### Article quality
How is your article unique? Tell us what makes your approach different from similar articles that have published on the same topic?
Is yours more in-depth? Does it cover additional topics? Do you provide handy tips or anecdotal advice?

#### References
N/A
### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)

### PR DESCRIPTION
**Pull request by Brian3811**
## Our **Peer Reviewer and Content Moderator teams do NOT provide any revisions services.** All revisions and edits should be completed on your own forked repo (as to not take up room in the queue) in order for our team to review them all in a timely manner. 

**ANY ARTICLE SUBMITTED WITH GLARING ERRORS WILL BE IMMEDIATELY CLOSED.**

>As a rule of thumb - please be sure to only submit articles (pull requests) that are fully polished and ready to be published. Be sure to go through our resources documents for extra and 3rd party (vetted) resources to help improve overall technical writing.

> If you have not already, please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve your overall technical writing so reviews are swifter, increase the chance of a payout and provide more value to our readers.

> For first-time contributors and for more details on our submission guidelines, **see our [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**.

#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [ x] Have you included a hero image that is **600x400 pixels**, under **300KB in size**, and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

This closes #4567

> Finally, delete the article checklist notes in blockquotes and submit your PR. We look forward to reviewing your article.

### CODE REVIEW

**Comment by Brian3811** [comment]
hello @paulodhiambo changes requested are in place

**Comment by hectorkambow** [comment]
@paulodhiambo  

If any other edits are needed - we will be closing the PR.

**Comment by louisefindlay23** [comment]
Plagiarism check complete

File: content/articles/introduction-to-kernel-methods-in-machine-learning/fixed index.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+  - url: /engineering-education/introduction-to-kernel-methods-in-machine-learning/hero.png
+    alt: Introduction to Kernel Methods in Machine Learning 
+---
+
+### Prerequisites
+- The Reader should have basic knowledge of machine learning. [Click here](https://www.digitalocean.com/community/tutorials/an-introduction-to-machine-learning) to read more about machine learning basics.
+- The reader should understand basic terms like Support Vector Machine, classifications, unsupervised learning, and supervised learning.
+- It's necessary for the reader to understand SVMs since kernels are implemented in SVM models. 
+### Introduction
```
Comment (Jethro-magaji) [inline content/articles/introduction-to-kernel-methods-in-machine-learning/fixed index.md:22]: "Introduction" should come before the "Prerequisite"

Add a "Table of content" section.

File: content/articles/introduction-to-kernel-methods-in-machine-learning/fixed index.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+The concept of "data drawn from some probability distribution" underpins many statistics and machine learning techniques. A probability distribution is a function defining all possible values and probabilities for a random variable at a specified range. The parameters of the underlying distribution are used to define parametric methods (or algorithms).
+
+However, the assumed model for the likelihood may not be correct; the "actual" distribution may be skewed, multimodal, with different components, and so on. In general, the scientist analyzing the data is unaware of this. This article aims to demonstrate kernels' fundamental concepts and their applications in statistics and machine learning.
+
+### _What are kernels?_
```
Comment (Jethro-magaji) [inline content/articles/introduction-to-kernel-methods-in-machine-learning/fixed index.md:27]: There's no need to italicize this heading.

File: content/articles/introduction-to-kernel-methods-in-machine-learning/fixed index.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+s1, ..., sp, ..., sk
+, sp ∈ sk − 1 
+
+such as: sp =1np 
+
+np∑i = 1 be (1.6)
+
+where **np** is the number of points assigned to cluster p during the initiation step by KSC.
+
+### Conclusion
```
Comment (Jethro-magaji) [inline content/articles/introduction-to-kernel-methods-in-machine-learning/fixed index.md:95]: Summarizing the tasks the audience just completed.

**Review by Jethro-magaji — CHANGES_REQUESTED** [review]

**Comment by Brian3811** [comment]
hello @Jethro-magaji  changes requested are in place

**Comment by Jethro-magaji** [comment]
Hi! @Brian3811 
Please change the current hero image to a high resolution free stock image of 600x400 PX.
Change the PR branch name to something readable, like this "introduction-to-kernel-methods-in-machine-learning"

File: content/articles/index.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+### _What are kernels?_
+Kernels, also known as kernel techniques or kernel functions, are a collection of distinct forms of pattern analysis algorithms, using a linear classifier, they solve an existing non-linear problem. SVM (Support Vector Machines) uses Kernels Methods in ML to solve classification and regression issues. The SVM (Support Vector Machine) employs ["Kernel Trick"](https://towardsdatascience.com/the-kernel-trick-c98cdbcaeb3f) where data is processed, and an optimal boundary for the various outputs is determined.
+
+In other words, a kernel is a term used to describe applying linear classifiers to non-linear problems by mapping non-linear data onto a higher-dimensional space without having to visit or understand that higher-dimensional region.
+
+### Unsupervised and supervised machine learning
+___Supervised machine learning___ is also known as supervised learning. It is distinguished by using labelled datasets to train algorithms that accurately classify data or predict outcomes.
```
Comment (Jethro-magaji) [inline content/articles/index.md:45]: You only mentioned supervised learning here, what about unsupervised learning?

**Review by Jethro-magaji — APPROVED** [review]
Hi @Brian3811 !
Well done for the awesome article, please take note of the reviews and edits next time you submit an article.
Thanks, overall it's a great article.

**Pull request merged**